README.md

```diff
@@ -17,8 +16,9 @@ machine_options = {
     :bootstrap_options => {
       :cloud_service_name => 'chefprovisioning',
       :storage_account_name => 'chefprovisioning',
-      #:vm_size => "A7"
-      :location => 'West US'
+      :vm_size => "Standard_D1"
+      :location => 'West US',
+      :tcp_endpoints => '80:80'
```

Comment:
Only some of them, the original driver code selects a default location and machine image (e.g. West US and an Ubuntu image) however this doesn't scale.  Updating the readme with the required parameters.